### PR TITLE
Foods

### PR DESCRIPTION
**Pull request by podoglyph**
This PR completes user stories for creating, validating, clearing, and persisting new foods on the foods.html page. It adds a static method in food.js for validation, which is called on form submission.

The validation itself, being for only two inputs, is simple if, else statement. Do you have a better suggestion for that? If more inputs were added, a loop to check for empty fields might be a better option. Thoughts?

### CODE REVIEW

File: lib/events.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ $(document).ready(function() {
         })
         .then(function(foods) {
             foods.forEach(function(food) {
-                $foodIndex.append(food.toHTML())
+                $foodIndex.prepend(food.toHTML())
```
Comment (Benjaminpjacobs) [inline lib/events.js:42]: nice I did this on my end too

File: lib/food.js [hunk 2]
```diff
@@ -42,10 +42,20 @@ class Food {
...
             .then(function(food) {
-                node.append(food.toHTML())
-
+                node.prepend(food.toHTML())
             })
     }
+
+    static validate(food) {
+    let msg = 'Please enter a';
+      if(food.name == '') {
```
Comment (Benjaminpjacobs) [inline lib/food.js]: lets try to stick to === because JS can be super weird about ==

File: lib/events.js [hunk 3]
```diff
@@ -97,7 +97,12 @@ $(document).ready(function() {
             name: $newFoodName.val(),
             calories: $newFoodCalories.val()
         }
-        Food.addNewFood(food, $foodIndex)
+        if(Food.validate(food)) {
+          Food.addNewFood(food, $foodIndex);
+          $newFoodName.val('');
+          $newFoodCalories.val('');
```
Comment (case-eee) [inline lib/events.js:103]: you could break out clearing the fields into a smaller function so you could reuse this in other places (if you need it of course) 👍

File: lib/food.js [hunk 4]
```diff
@@ -42,10 +42,20 @@ class Food {
...
             .then(function(food) {
-                node.append(food.toHTML())
-
+                node.prepend(food.toHTML())
             })
     }
+
+    static validate(food) {
+    let msg = 'Please enter a';
+      if(food.name === '') {
```
Comment (case-eee) [inline lib/food.js:51]: yup, i'd go with a conditional here for now, then move that to a loop if you have more than just a few fields to validate

**Review by case-eee — COMMENTED** [review]
@podoglyph @Benjaminpjacobs let me know if you have questions on my feedback - i'd also suggest using HTML5 validations as well as an added layer of protection 👍

**Pull request merged**